external_data/models/fund.py

```diff
@@ -0,0 +1,25 @@
+from dataclasses import dataclass
+from typing import List
+
+from external_data.models.round import Round
+
+
+@dataclass
```

Comment:
Lovely that you're using dataclass....check out attrs - its already included in our requirements by other libraries and _some_ people think it's better then dataclass. It's just slightly cooler and has more batteries included.

Reply:
Noted. Maybe we need some use cases where our classes are doing more hard work, validation etc so we can get out the big guns?